Manito/Manito/Presentation/Scene/SplashScene/ViewController/SplashViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  SplashViewController.swift
+//  Manito
+//
+//  Created by SHIN YOON AH on 2022/06/16.
+//
+
+import Combine
+import UIKit
+
+final class SplashViewController: UIViewController {
+    
+    // MARK: - ui component
+    
+    private let splashView: SplashView = SplashView()
+    
+    // MARK: - property
+    
+    private var cancelBag: Set<AnyCancellable> = Set()
+
+    private let viewModel: any BaseViewModelType
+
+    // MARK: - init
+    
+    init(viewModel: any BaseViewModelType) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - life cycle
+    
+    override func loadView() {
+        self.view = self.splashView
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.bindViewModel()
+    }
+
+    // MARK: - func
+    
+    private func bindViewModel() {
+        let output = self.transformedOutput()
+        self.bindOutputToViewModel(output)
+    }
+
+    private func transformedOutput() -> SplashViewModel.Output? {
+        guard let viewModel = self.viewModel as? SplashViewModel else { return nil }
+        let input = SplashViewModel.Input(
+            viewDidLoad: self.viewDidLoadPublisher
+        )
+        return viewModel.transform(from: input)
+    }
+
+    private func bindOutputToViewModel(_ output: SplashViewModel.Output?) {
+        guard let output = output else { return }
```

Comment:
p4;

```suggestion
        guard let output else { return }
```

이렇게 써도 될 것 같습니다!

Reply:
넵! 확인했습니다 🫡